File: Java/1. Two Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> numIndex = new HashMap<>();
        int[] res = new int[2];
        for (int i = 0; i < nums.length; i++) {
            if (numIndex.containsKey(target - nums[i])) {
                res[0] = numIndex.get(target - nums[i]);
                res[1] = i;
                break;
            }
            numIndex.put(nums[i], i);
        }
        return res;
    }
}

File: Python/1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        store = dict()
        for i in range(0, len(nums)):
            if target - nums[i] in store:
                return [store[target - nums[i]], i]
            store[nums[i]] = i